app/controllers/reports_controller.rb

```diff
@@ -20,8 +22,28 @@ def create
     end
   end
 
+  def update_dropdown
+    org_ids = params[:org_ids]
+
+    @protocols = []
+    if org_ids.length > 1 
+      org_ids.each do |org_id|
+        @protocols << find_protocols(org_id)
+      end
+    else
+      @protocols << find_protocols(org_ids)
+    end
+    @protocols = @protocols.flatten
+  end
+
   private
 
+  def find_protocols(org_ids)
```

Comment:
You could create a method in the Protocol class that does this action. 

     #protocol.rb
     
       def self.method_name
         joins(:sub_service_request).where(sub_service_requests: { organization_id: org_ids })
       end

Then in find_protocols

       def find_protocols(org_ids)
         Protocol.method_name
       end

Benefit here is that you could reuse this method elsewhere